duckduckgo/content-scope-scripts
# ntp: prepare translations

**Asana Task/Github Issue:** https://app.asana.com/0/1201141132935289/1208831856736789/f

## Description

- [x] split the translations into separate `strings.json` files
- [x] combine them during the build process, so that we still get a single output file in `src/locales/en/new-tab.json`
  - this file is the one we then submit to smartling
- [x] I applied the ship review feedback around copy
- [x] I improved the notes for translators 

## Testing Steps

- Check the text strings are outputting correctly here https://deploy-preview-1262--content-scope-scripts.netlify.app/build/pages/new-tab/
- A follow-up PR will improve the styling

## Checklist


*Please tick all that apply:*
